tst/CTA.Rules.Test/Models/ExpectedOutputConstants.cs

```diff
@@ -1651,7 +1651,7 @@ public static void Main(string[] args)
 @"using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.Hosting;
 
-namespace ASP.NET-WebForms
+namespace ASP.NETWebForms
```

Comment:
Once my pr that changes namespace hyphens to underscores is merged this would actually be ASP.NET_WebForms. Might want to change it to that then just merge it after mine goes through